x-pack/test/functional/apps/discover/value_suggestions.ts

```diff
@@ -49,6 +53,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       beforeEach(async () => {
+        await PageObjects.common.setTime({ from, to });
```

Comment:
@rashmivkulkarni are you sure you want this in a `beforeEach` instead of a `before`?

Reply:
yes , moved it to `before`